include/z64save.h

```diff
@@ -524,7 +524,7 @@ typedef enum LinkAge {
 #define EVENTCHKINF_4C 0x4C
 #define EVENTCHKINF_CREATED_RAINBOW_BRIDGE 0x4D
 #define EVENTCHKINF_CAUGHT_BY_CASTLE_GUARDS 0x4E // set but unused
-#define EVENTCHKINF_REVEAL_MASTER_SWORD_CS 0x4F // Cutscene in Temple of Time as adult after pulling the Master Sword for the first time
+#define EVENTCHKINF_REVEALED_MASTER_SWORD 0x4F // Cutscene in Temple of Time as adult after pulling the Master Sword for the first time
```

Comment:
Im so sorry, I just noticed this comment is wrong lol. If you dont want to fix it I can do it and push to your branch.

Reply:
no worries lol, I should've notice it

Reply:
also small note: this cutscene can play as child and as adult iirc, hence why I didn't say "as child" here